### PR TITLE
Remove the explicit setting of a pygments theme

### PR DESCRIPTION
**Pull request by pradyunsg**
This allows the underlying theme being used to provide the pygments theme instead which looks better (subjectively).

# Description

# Checklist

* [x] Make sure changes are covered by existing or new tests.
* [x] For at least one Python version, make sure local test run is green.
* [x] Create a file in `src/towncrier/newsfragments/`. Describe your
  change and include important information. Your change will be included in the public release notes.
* [x] Make sure all GitHub Actions checks are green (they are automatically checking all of the above).
* [x] Ensure `docs/tutorial.rst` is still up-to-date.
* [x] If you add new **CLI arguments** (or change the meaning of existing ones), make sure `docs/cli.rst` reflects those changes.
* [x] If you add new **configuration options** (or change the meaning of existing ones), make sure `docs/configuration.rst` reflects those changes.

### CODE REVIEW

**Comment by pradyunsg** [comment]
> * Create a file in `src/towncrier/newsfragments/`.

I don't think this is a news worthy change, but let me know if you want one.

File: docs/conf.py [hunk 1]
```diff
@@ -75,7 +75,7 @@
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
+# pygments_style = "sphinx"
```
Comment (adiroiban) [inline docs/conf.py]: Thanks for the update.
Why not just remove these 2 lines?

Is there any value in leaving them commented?

**Comment by adiroiban** [comment]
Thanks for the update.

I triggerd the build so can check[ the new version on Read the docs.](https://towncrier--625.org.readthedocs.build/en/625/markdown.html)

> I don't think this is a news worthy change, but let me know if you want one.

If it's not news worthy, then we need a `newsfragment/625.misc` file.

This confirm that the release notes information was considred and it was decided not to add any explicit text.

As opposed of not having a newsfragment file and thinking that the author of the PR has forgot about the release notes :)

File: docs/conf.py (hunk 1 above)
Comment (pradyunsg) [reply docs/conf.py]: That's the format of the conf.py file today -- with a bunch of unused commented out configuration variables. While I would strongly encourage doing something like https://github.com/pytest-dev/pytest/commit/05979bc4a97491c8b75f8baf95ec16f5b5516353, I didn't have much interaction with the maintainers prior to this and did not want to show up with a giant PR that changes everything. 😅

Comment (pradyunsg) [reply docs/conf.py]: Actually, I take that back. There's only a tiny bit of code at the top of the file -- Lemme remove this.

**Comment by adiroiban** [comment]
This is subjective ... but I think that we can merge it.

Thanks :)

**Pull request merged**